myft/ui-instant.js

```diff
@@ -0,0 +1,121 @@
+/*
+* ui for instant alerts buttons
+* ui.js has become so complex that we are splitting instant alerts into its own file until it gets refactored
+*/
+
+const nextButtons = require('../buttons');
+const nNotification = require('../notification');
+const Delegate = require('ftdomdelegate');
+const myftClient = require('next-myft-client');
+
+const delegate = new Delegate(document.body);
+const $ = require('../utils').$
+const $$ = require('../utils').$$
+const subscribeUrl = '/products?segID=400863&segmentID=190b4443-dc03-bd53-e79b-b4b6fbd04e64';
+const signInLink = '/login';
+
+const UI_HOOK = '[data-myft-ui="instant"]';
+let config;
+
+// until API is updated to return modelled response data from create calls, fallback to old raw format
+function apiBackwardsCompatibility (response) {
+	return (response._rel) ? response._rel.instant : response.results[0].rel.properties.instant;
+}
+
+function updateConceptData (actorId, subjectId, data) {
+	myftClient.updateRelationship('user', actorId, 'followed', 'concept', subjectId, data);
```

Comment:
To avoid passing `actorId` and fetching from the DOM you can pass `null` here and the client will use the memoized uuid it has from the initial init.